packages/styling/components/alert.scss

```diff
@@ -0,0 +1,81 @@
+@import "../variables";
+@import "../icons";
+
+$dso-alert-icon-size: $u4;
+$dso-alert-horizontal-padding: $u2;
+$dso-alert-vertical-padding: $u1 * 2.5 - 1px;
+$dso-alert-left-padding: 2 * $dso-alert-horizontal-padding + $dso-alert-icon-size - 1px; // 1px border
+
+$alert-success-bg:            $state-success-bg;
```

Comment:
Waarom nog steeds bootstrap indenting?

Reply:
Goeie vraag, die wilde ik nog aankaarten.

Niet doen dus?

Reply:
Mijn mening zegt inderdaad "niet doen". We doen het verder nergens.

Reply:
Ok, ik heb ze overal weggehaald behalve bij `@dso-toolkit/styling/variables/colors.scss`, daar vind ik het echt een meerwaarde hebben